### ISSUE TITLE
Podfile.lock for 5.1.2 references the latest commit for version 6 RC code in the repository.

### ISSUE THREAD

**Issue by chrispaynter** [FormidableLabs/react-native-app-auth#552]
## Issue

Podspec for npm version 5.1.2 doesn't specify a source tag. When running pod install, the checkout options commit in the pod file.lock points to the latest repo changes, which are version 6 rc specific.

The result is that when running the authorise process, we get this error:

![image](https://user-images.githubusercontent.com/1684083/89902226-dc1fea80-dbee-11ea-91a3-1a6b15d6917b.png)

```
RNAppAuth.authorize was called with 9 arguments but expects 10 arguments.
```

Reverting checkout options in pod file.lock to the [5.1.2](https://github.com/FormidableLabs/react-native-app-auth/releases/tag/v5.1.2) commit `a8c4d47` resolves the issue.

```
CHECKOUT OPTIONS:
  react-native-app-auth:
    :commit: a8c4d474e49fffafebed2f168384599b344ae214
    :git: https://github.com/FormidableLabs/react-native-app-auth.git
```

**NOTE:**
In order to get swift files to compile correctly, I had to manually add the podspect to my Podfile. Not sure if this could be a cause of the above issue.

```
  pod 'react-native-app-auth', :podspec => '../node_modules/react-native-app-auth/react-native-app-auth.podspec'
```

Disclaimer - I don't have the deepest understanding of podfile mechanics.

**Comment by kadikraman** [comment]
Hi @chrispaynter - thanks for pointing this out!

I just tried this on a fresh project, and it works as expected when autolinking, but I do get the same error when adding the podspec. Looks like you're right, our podscpec should be linking to the specific version tag. I guess it works with autolinking because it'd read the `react-native-app-auth` code from `node_modules` instead of trying to fetch it again from source, which is why we never spotted the issue.

Replacing the source line in the Podspec with
```
s.source       = { :git => 'https://github.com/FormidableLabs/react-native-app-auth.git', :tag => "v#{s.version}" }
```
seems to resolve it.

**Comment by chrispaynter** [comment]
Thanks for fixing this quickly @kadikraman!

Will this be release as a patch update for 5.1 on NPM soon?

**Comment by kadikraman** [comment]
Released in [v5.1.3](https://github.com/FormidableLabs/react-native-app-auth/releases/tag/v5.1.3) 🎉

**Comment by chrispaynter** [comment]
Cracking. Thanks for that!
